apps/vercel/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,118 @@
+import { useCallback, useState, useEffect, ChangeEvent } from 'react';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import {
+  Box,
+  Flex,
+  FormControl,
+  Heading,
+  HelpText,
+  Paragraph,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { styles } from '../components/config-screen/ConfigScreen.styles';
+import VercelIcon from '../components/common/VercelIcon';
+
+export interface AppInstallationParameters {
+  accessToken?: string;
```

Comment:
Instead of optional fields, I feel we could make sure everything is always defined. This will lead to a nicer dev experience as we won't have to null check when trying to access properties on `parameters`.

When initializing the AppInstallation parameters we can instantiate it with a default object. 
```ts
const [parameters, setParameters] = useState<AppInstallationParameters>({
  accessToken: ''
});
```

Reply:
yeah sounds good 👍 will address this tomorrow

Reply:
With the OAUTH flow, will we be removing this field? If so then I agree, completely a moot point!